controller/server/gui.ml

```diff
@@ -404,7 +404,19 @@ module ChangelogGui = struct
         Lwt.return (page (Changelog_page.html changelog)))
 end
 
-let routes ~shutdown ~health_s ~update_s ~rauc ~connman app =
+module RemoteManagementGui = struct
+  let build ~systemd app =
+    app
+    |> post "/remote-management/enable" (fun _ ->
+        let%lwt _ = Systemd.Manager.start_unit systemd "zerotierone.service" in
+        Lwt.return (success (Format.sprintf "Starting remote management service.")))
```

Comment:
Could we forward directly back to the info page instead of a separate status message, or does this lead to problems if zerotier starts slowly? I guess we could delay the response for some milliseconds to give time for the service to start.

Reply:
Disabling does not seem to require a delay: https://github.com/dividat/playos/pull/65/commits/233d170633e5200105c33fbc595fe70ba77abadc.